### ISSUE TITLE
Recommendations for dependencies without version

### ISSUE THREAD

**Issue by norgeindian** [yireo/Yireo_ExtensionChecker#27]
When a dependency in `composer.json` is missing, which does not have any version, the following output is generated:
```
| No composer dependency found for "ext-json"   | Missing composer dependency | Current version is *. Perhaps use ~*?       |
```
As far as I know, `~*` is not a valid version number, is it?
So I assume, the output is wrongly created, and it should look different for `*`.

**Comment by jissereitsma** [comment]
Yes, this is definitely wrong. The version `*` is there when fetching it for PHP and PHP extensions, so then no suggestion is needed. This is partially already fixed in https://github.com/yireo/Yireo_ExtensionChecker/commit/071a0f6bf72e13ab41e3ce12ffb25e9cacd76379 but the usage of that new function is underway. It will be fixed in the next version.

**Comment by jissereitsma** [comment]
This should now be fixed in 2.1.2.
